pkg/asset/installconfig/openstack/validation/platform.go

```diff
@@ -23,9 +23,6 @@ func ValidatePlatform(p *openstack.Platform, n *types.Networking, ci *CloudInfo)
 	// validate the externalNetwork
 	allErrs = append(allErrs, validateExternalNetwork(p, ci, fldPath)...)
 
-	// validate platform flavor
-	allErrs = append(allErrs, validateFlavor(p.FlavorName, ci, ctrlPlaneFlavorMinimums, fldPath.Child("computeFlavor"), true)...)
```

Comment:
FYI I talked to @iamemilio today and we might re-introduce that validation somewhere, for backward compatibility.
Since we still support that parameter, I think we still need to validate it, but we haven't figured out how yet; waiting on this PR to merge first to re-iterate on that topic.

Therefore, I have no strong opinion if whether or not we should remove that validation in that PR or just leave it here and we'll deal with it later with Emilio.

Reply:
Validation is done after the up-conversion. There is no need to validate the deprecated field.

Reply:
this will be done into a different PR: https://github.com/openshift/installer/pull/4383 for easier backport to 4.6.